omnibus/resources/chef-workstation/msi/source.wxs.erb

```diff
@@ -51,144 +51,204 @@
-                  <RegistryValue Type="string" Name="RuntimeVersion" Value="v4.0.30319" />
-                  <RegistryValue Type="string" Name="CodeBase" Value="[PSMODULES]chef\Chef.PowerShell.dll" />
-                </RegistryKey>
-                <RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}\InprocServer32\1.0.0.0">
-                  <RegistryValue Type="string" Name="Class" Value="Chef.PowerShell" />
-                  <RegistryValue Type="string" Name="Assembly" Value="Chef.PowerShell, Version=1.0.14.0, Culture=neutral, PublicKeyToken=7def9f799d039a95" />
-                  <RegistryValue Type="string" Name="RuntimeVersion" Value="v4.0.30319" />
-                  <RegistryValue Type="string" Name="CodeBase" Value="[PSMODULES]chef\Chef.PowerShell.dll" />
-                </RegistryKey>
-                <RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}\ProgId">
-                  <RegistryValue Type="string" Value="Chef.PowerShell" />
-                </RegistryKey>
-                <RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}\Implemented Categories\{62C8FE65-4EBB-45E7-B440-6E39B2CDBF29}" />
-              </Component>
-            </Directory>
-            <Directory Id="EMBEDDED" Name="embedded" >
-              <Directory Id="EMBEDDEDBIN" Name="bin" >
-                <Component Id="ChefWSEnvHacks" Guid="{3498A65D-33D8-4CF2-BAB8-6FA072A7070D}" >
-                  <Environment Id="EnvHacksEnvironment"
-                               Name="CHEFDK_ENV_FIX" Action="set" System="yes" Value="1" />
-                </Component>
-              </Directory>
-            </Directory>
-          </Directory>
-        </Directory>
-      </Directory>
+  <% end %>
+
+  <!-- TODO
+       For now, using COMMON_ALTSTARTUP which is the global auto-start folder. Note that this
+       may cause some trouble - we haven't yet looked into how the App should behave when multiple
+       instances are running. We can instead use WIXDIR_ALTSTARTUP which is per-user - but that means
+       if it's installed on a shared server, only one user will get the tray app be default.
+	-->
+	<PropertyRef Id="WIX_DIR_COMMON_ALTSTARTUP" />
+
+	<CustomActionRef Id="WixBroadcastSettingChange" />
+	<CustomActionRef Id="WixBroadcastEnvironmentChange" />
+
+	<SetDirectory Id="WINDOWSVOLUME" Value="[WindowsVolume]"/>
+	<Directory Id="TARGETDIR" Name="SourceDir">
+		<Directory Id="WINDOWSVOLUME">
+			<Directory Id="INSTALLLOCATION" Name="opscode">
+				<Directory Id="PROJECTLOCATION" Name="chef-workstation">
+					<Directory Id="PROJECTLOCATIONBIN" Name="bin">
+						<Component Id="ChefWSPath" Guid="{BC744260-FD5B-4A0F-8EE5-76283072CC09}" >
+							<Environment Id="Environment"
+								Name="PATH" Action="set" Part="last" System="yes" Value="[PROJECTLOCATIONBIN]" />
+						</Component>
+						<Component Id="StartChefWSPowershellShortcut" Guid="{69EF5736-ED67-45B3-887A-FBFC0AB6DA13}" >
```

Comment:
Nix me if this is pedantic. But could we refer to this as `ChefWsApp` and `start-chef-ws-app.ps1` in this component to be most clear?

Reply:
This isn't for the app - this is the config for the powershell shortcut. I can rename the component itself though, "start-ws.ps1" isn't really "Start Chef Workstation". `chef-ws-powershell.ps1`? 

I will do this in a separate minor PR to avoid more rebasing of downstream branches.